content/docs/setup/kubernetes/quick-start-ibm/index.md

```diff
@@ -69,69 +46,99 @@ Make sure to use the `kubectl` CLI version that matches the Kubernetes version o
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Add the IBM Cloud Helm repository to your Helm instance:
+### Deploy the Istio Helm charts
+
+1. Install the `istio-init` chart to bootstrap all the Istio CRDs:
 
     {{< text bash >}}
-    $ helm repo add ibm-charts https://registry.bluemix.net/helm/ibm-charts
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
     {{< /text >}}
 
-### Deploy the Istio Helm chart
-
-1. If using a Helm version prior to 2.10.0, install Istio’s Custom Resource Definitions via `kubectl apply`, and wait a few seconds for the CRDs to be committed
-to the Kubernetes API server:
+    To verify that all Istio CRDs were committed in the Kubernetes api-server, check the number of CRDs with the specified suffix with the following command. Verify that the number of total CRDs created was `58` for Istio:
```

Comment:
I don't understand what CRDs with the specified suffix means. What suffix is it talking about? Why not just one simple sentence like this:

Verify that all `58` Istio CRDs were committed to the Kubernetes api-server using the following command:

Reply:
I admit to pulling that section from the base install documentation. I believe the original author is describing the `grep` piece of the command. The command gets all the CRDS, greps those with the specific suffix, and then gets the count of lines.

That said, the suggested text is more concise.

Done.

Reply:
It greps, but not for any suffix. Maybe it could say "with one of the substrings as specified in the following grep command". Then it would be correct, but way too verbose.

I notice that this verify step is also in the install with helm instructions (2 places), although the wording there is a little different (correct), and also in the minimal install instructions.

I suggest you change it to the short version that I suggested in all 4 places and then when we merge this PR it will be the same everywhere. It should probably be moved to a fragment file so that it doesn't need to be copied everywhere, but I think that's a cleanup exercise that can be done later.

Reply:
Updated so all 4 places match suggested text.